packages/auth/src/core/auth.ts

```diff
@@ -54,7 +54,8 @@ export class ThirdwebAuth {
       nonce: parsedOptions?.nonce,
       expiration_time:
         parsedOptions?.expirationTime || new Date(Date.now() + 1000 * 60 * 5),
```

Comment:
uhh shouldn't this be 24h instead of 5min?

Reply:
no